### PR TITLE
Sales receipt var

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
Changes in this branch include:

- Included missing `customer_id` fields within the `purchase_line` models.
- Addition of purchase order models which have no downstream effects and are not used by many customers. As such, the variable for these models has by default been set to `false`.
- Addition of the `using_sales_receipt` variable for customers to disable the sales_receipt models if they are not used.
- Staging model documentation fix. Incorrect reference to `salesforce` changed to `quickbooks`.
- Temporary change of branch for `dbt_fivetran_utils` package reference. Once the [PR](https://github.com/fivetran/dbt_fivetran_utils/pull/13) for `dbt_fivetran_utils` is approved this will be removed.

[Joe PR Loom Description](https://www.loom.com/share/5dabc17bcfa941d4867063aacc988608)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,15 +54,18 @@ vars:
     using_invoice_bundle: false         #disable if you don't have invoice bundles in QuickBooks
     using_journal_entry:  false         #disable if you don't have journal entries in QuickBooks
     using_payment:        false         #disable if you don't have payments in QuickBooks
+    using_purchase_order: false         #disable if you don't have purchase orders in QuickBooks
     using_refund_receipt: false         #disable if you don't have refund receipts in QuickBooks
     using_transfer:       false         #disable if you don't have transfers in QuickBooks
     using_vendor_credit:  false         #disable if you don't have vendor credits in QuickBooks
+    using_sales_receipt:  false         #disable if you don't have sales receipts in QuickBooks
+    using_purchase_order: true          #enable if you want to include purchase orders in your staging models
```
Comment (fivetran-kristin) [inline README.md:62]: duplicated

Comment (fivetran-kristin) [reply README.md:62]: I actually think the top one is duplicated.  The `using_purchase_order` variable should be set to `false` by default, so our instructions should show users how to do the opposite.

File: packages.yml [hunk 2]
```diff
@@ -3,4 +3,5 @@ packages:
     version: [">=0.6.0", "<0.7.0"]
 
   - git: https://github.com/fivetran/dbt_fivetran_utils.git
+    revision: enabled_vars_one_true   #To remove after sales receipt test
```
Comment (fivetran-kristin) [inline packages.yml:6]: I've approved the PR in fivetran_utils so this can be pinned back to master.

**Review by fivetran-kristin — APPROVED** [review]
LGTM -- There are two updates that need to be made prior to merging (duplicate variable in the dbt_project.yml file and the fivetran-utils dependency). Feel free to merge after making those changes, thanks!

File: README.md (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply README.md:61]: Removed the duplicate. `using_purchase_order` is by default set to `false`. The readme now shows to enable as `true` if it is to be used.

File: packages.yml (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply packages.yml]: Updated to only install master branch

**Pull request merged**